etc/PrimarySourceSetDataModelSample.json

```diff
@@ -43,7 +43,13 @@
     }
   ],
   "dct:temporal": [
-    { "name": "Colonial America" }
+    { 
+      "@id": "ushist:united-states-era-2",
+      "@type": "dct:PeriodOfTime",
```

Comment:
> But I think that edm:begin/end would be better.

:+1: 

`start/end` are referred to in that document as XML attributes. They aren't defined in DC Elements or Terms namespaces. Does make me curious what DCMI's guidance would be on this front, though.